src/MFRC522.h

```diff
@@ -360,6 +360,12 @@ class MFRC522 {
 	bool PCD_PerformSelfTest();
 	
 	/////////////////////////////////////////////////////////////////////////////////////
+	// Power control functions
+	/////////////////////////////////////////////////////////////////////////////////////
+	void power_down();
```

Comment:
Naming like the other functions and more clear. I suggest `void PCD_SoftPowerDown();`